journal/src/main/scala/com/evolutiongaming/kafka/journal/JsonCodec.scala

```diff
@@ -89,51 +89,77 @@ object JsonCodec {
   }
 
 
-  final case class Decode[F[_]](fromBytes: FromBytes[F, JsValue])
+  final case class Decode[F[_]](fromBytes: FromBytes[F, JsValue], fromStr: String => F[JsValue])
```

Comment:
why do we need `fromStr` at all here?

Reply:
To cover parsing strings.

 https://github.com/evolution-gaming/kafka-journal/pull/136/files#diff-a2d895d6800c92630153ca061a462ad3R89

Reply:
but why should we cover parsing strings at all ?